templates/monitoring/datadog.tf.tpl

```diff
@@ -1,4 +1,5 @@
 {{- $_ := stencil.ApplyTemplate "skipIfNotService" -}}
+{{- $_ := file.Static }} 
```

Comment:
I don't think we want the entire Datadog monitoring file to be user-managed. This would dramatically reduce the ability of us to roll out new features.

I was thinking more along the lines of:
* SLO block for people to add custom SLOs.
* boolean to opt-out of default SLOs.